pkg/kubernetes/listing.js

```diff
@@ -3,12 +3,135 @@ define([
     "base1/cockpit",
     "base1/angular",
     "base1/term",
+    "kubernetes/client",
     "kubernetes/app"
-], function($, cockpit, angular, Terminal) {
+], function($, cockpit, angular, Terminal, kubernetes) {
     'use strict';
 
     var _ = cockpit.gettext;
 
+    var k8client =  kubernetes.k8client();
+    var adjust_btn = $('#adjust-entity-button'); 
+    var adjust_entity_dlg = $('#adjust-entity-dialog');
+    
+    var delete_pod_dlg = $('#delete-pod-dialog');
+    var delete_pod_btn = $('#delete-pod-button');
+
+
+    adjust_entity_dlg.on('show.bs.modal', function(e) {
+        var rc = $(e.relatedTarget).attr("data-entity");
+        var rc_json = JSON.parse(rc);
+        $(e.currentTarget).find('input[id="replica-count"]').val(rc_json.spec.replicas);
+
+        adjust_btn.on('click', function() {
+            function resize(item, value) {
+                var spec = item.spec;
+                if (!spec) {
+                    console.warn("replicationcontroller without spec");
+                    return false;
+                }
+
+                /* Already set at same value */
+                if (spec.replicas === value)
+                    return false;
+
+                spec.replicas = value;
+                return true;
+            }
+
+            function gather(input, rc) {
```

Comment:
There is only one task, why do we have to have a gather function that returns a set of task functions?